tools/main-transpile-wrapper.js

```diff
@@ -0,0 +1,8 @@
+// This file exists because it is the file in the tool that is not automatically
+// transpiled by Babel.
+
+require('source-map-support').install();
+require('babel/register')(require('./babel-config.js')); // #RemoveInProd this line is removed in isopack.js
```

Comment:
I set it to cache inside `.meteor/babel-cache.json`, is that good? Not sure if we want `.meteor` to be identical to the install or something.